packages/auth/android/src/main/java/io/invertase/firebase/auth/ReactNativeFirebaseAuthModule.java

```diff
@@ -245,13 +245,14 @@ public void setAutoRetrievedSmsCodeForPhoneNumber(
 
   /**
    * Disable app verification for the running of tests
+   *
    * @param appName
    * @param disabled
    * @param promise
    */
   @ReactMethod
   public void setAppVerificationDisabledForTesting(
-    String appName, boolean disabled, Promise promise) {
+      String appName, boolean disabled, Promise promise) {
```

Comment:
I think running the files through `yarn lint:android` will just auto-fix this and get that lint check to pass after you commit the result?

Reply:
Yes, I just ran linter and that was auto-fixed. But it's not an intended results as I expected it to fix my additions only. Perhaps revert this change would be more desirable?

Reply:
I just pushed the result of `yarn lint:android` to fix the lint CI error, reviewing through the PR with it pulled locally right now